opendata_module/opmon_opendata/api/helpers.py

```diff
@@ -0,0 +1,213 @@
+import json
+from datetime import datetime
+from gzip import GzipFile
+from io import BytesIO
+from typing import Generator, List, Optional, Tuple, TypedDict
```

Comment:
```suggestion
from datetime import datetime
from gzip import GzipFile
from io import BytesIO
import json
from typing import Generator, List, Optional, Tuple, TypedDict
```

Reply:
Good catch! I used https://pypi.org/project/isort/ to sort imports, but strangely, this util does not correct import order.